_includes/google-search.html

```diff
@@ -0,0 +1,12 @@
+<script>
+    (function() {
+      var cx = 'e20a3e7ce37c2454c';
+      var gcse = document.createElement('script');
+      gcse.type = 'text/javascript';
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#type :

> Authors are encouraged to omit the attribute if the script refers to JavaScript code rather than specify a MIME type.